web/src/app/app.component.html

```diff
@@ -65,7 +65,7 @@
         </mat-toolbar>
 
         <div class="content-container">
-            <mat-progress-bar *ngIf="showSpinner" mode="indeterminate"></mat-progress-bar>
+            <mat-progress-bar *ngIf="progress > 0" mode="buffer"></mat-progress-bar>
```

Comment:
Spinner looks very light . It does't make feel user that there is some activity going on in background.

Reply:
I tried changing it back to the `indeterminate` style but it looks odd because it it so fast it moves a small amount and disappears and then starts again from the beginning, so for now I will use the `buffer` style because it looks better - yes may need it tot stand out more though (I tried making it thicker but it looked strange), I will try changing the colour

Reply:
Please let me know once done so that I can review

Reply:
Sure will do 👍 . It will probably be tomorrow, I need to resolve some conflicts also now too 🤓